uint8 OK = 0
uint8 WARNING = 1
uint8 ERROR = 2

time stamp
uint16 index
uint8 status
uint16 errors_count
uint16 warns_count